spec/controllers/api/users_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Api::UsersController, type: :controller do
```

Comment:
Are you planning on writing tests?

Reply:
Yeah, I was planning to circle back and rewrite adding tests.